docs/spec/crosschain/messages.md

```diff
@@ -215,7 +215,7 @@ message MsgWhitelistERC20 {
 
 ## MsgUpdateTssAddress
 
-Authorized: admin policy group 2.
+UpdateTssAddress updates the TSS address.
```

Comment:
A task planned in the future, is to add a table that associate the messages with required policy, this will allow to have eveything documented in a central place, so it is fine to not keep it here